src/roadmap-goals.md

```diff
@@ -0,0 +1,119 @@
+# Roadmap & Goals
+
+the roadmap gives you an overview and orientation for your first weeks at VI.
+
+## Goal
+> Every new frontend developer is able to implement components in our Biotope projects, in compliance with our VI standards!
+
+## Roadmap
+
+The following part is an overview of all the topics and challanges, that you are going to pass the next view weeks.
+
+
+<roadmap-component>
+
+### Preparation
+
+You will be assigned a mentor and also buddies for each lesson.
+You have to organize your accounts and tools and afterwards this roadmap will be adjust to your individual skills. Beside that you will also get an introduction to the development manifesto.
+
+*Goal:you can start right away and know where you are heading without being over- or underchallenged*
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Getting started
+
+The first part of your journey is to get an overview of the topics, that come across during your first weeks. You will learn which vision the onboarding process follows and get in touch with our toolset like JIRA and GIT/BITBUCKET
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Fundamental I
```

Comment:
We have FundamentalS everywhere, so we should do it here too I guess

Reply:
fixed